recipes/snakemake/meta.yaml

```diff
@@ -48,6 +50,10 @@ test:
     - export GIT\_PYTHON_REFRESH=warn
     - snakemake --version
     - snakemake --version | grep "{{ version }}"
+    # Generate a report to check whether graphviz dependencies (libXau) are properly installed:
+    - 'printf %s\\n "rule empty:" > Snakefile && snakemake --cores 1 --report'
+    # Remove test artifacts from command above.
+    - rm -rf .snakemake Snakefile report.html
```

Comment:
The actual build succeeded, but then
```
Traceback (most recent call last):
  File "/home/circleci/project/miniconda/bin/bioconda-utils", line 8, in <module>
    sys.exit(main())
  File "/home/circleci/project/miniconda/lib/python3.7/site-packages/bioconda_utils/cli.py", line 971, in main
    bioconductor_skeleton, clean_cran_skeleton, autobump, bot
  File "/home/circleci/project/miniconda/lib/python3.7/site-packages/argh/dispatching.py", line 328, in dispatch_commands
    dispatch(parser, *args, **kwargs)
  File "/home/circleci/project/miniconda/lib/python3.7/site-packages/argh/dispatching.py", line 174, in dispatch
    for line in lines:
  File "/home/circleci/project/miniconda/lib/python3.7/site-packages/argh/dispatching.py", line 277, in _execute_command
    for line in result:
  File "/home/circleci/project/miniconda/lib/python3.7/site-packages/argh/dispatching.py", line 260, in _call
    result = function(*positional, **keywords)
  File "<boltons.funcutils.FunctionBuilder-5>", line 2, in build
  File "/home/circleci/project/miniconda/lib/python3.7/site-packages/bioconda_utils/cli.py", line 130, in wrapper
    func(*args, **kwargs)
  File "<boltons.funcutils.FunctionBuilder-4>", line 2, in build
  File "/home/circleci/project/miniconda/lib/python3.7/site-packages/bioconda_utils/cli.py", line 59, in wrapper
    func(*args, **kwargs)
  File "/home/circleci/project/miniconda/lib/python3.7/site-packages/bioconda_utils/cli.py", line 476, in build
    keep_old_work=keep_old_work)
  File "/home/circleci/project/miniconda/lib/python3.7/site-packages/bioconda_utils/build.py", line 343, in build_recipes
    linter=linter)
  File "/home/circleci/project/miniconda/lib/python3.7/site-packages/bioconda_utils/build.py", line 149, in build
    pkg_test.test_package(pkg_path, base_image=base_image)
  File "/home/circleci/project/miniconda/lib/python3.7/site-packages/bioconda_utils/pkg_test.py", line 170, in test_package
    p = utils.run(cmd, env=env, cwd=d, mask=False)
  File "/home/circleci/project/miniconda/lib/python3.7/tempfile.py", line 805, in __exit__
    self.cleanup()
  File "/home/circleci/project/miniconda/lib/python3.7/tempfile.py", line 809, in cleanup
    _shutil.rmtree(self.name)
  File "/home/circleci/project/miniconda/lib/python3.7/shutil.py", line 485, in rmtree
    _rmtree_safe_fd(fd, path, onerror)
  File "/home/circleci/project/miniconda/lib/python3.7/shutil.py", line 423, in _rmtree_safe_fd
    _rmtree_safe_fd(dirfd, fullname, onerror)
  File "/home/circleci/project/miniconda/lib/python3.7/shutil.py", line 427, in _rmtree_safe_fd
    onerror(os.rmdir, fullname, sys.exc_info())
  File "/home/circleci/project/miniconda/lib/python3.7/shutil.py", line 425, in _rmtree_safe_fd
    os.rmdir(entry.name, dir_fd=topfd)
PermissionError: [Errno 13] Permission denied: 'conda'

Exited with code exit status 1

CircleCI received exit code 1
```
happens. No idea why..

Reply:
Transient error?

Reply:
I'm not sure, actually. My hunch would be that it has to do with the "conda" user that the `condaforge/linux-anvil-comp7` has. But that's just a hunch.
I'll work on how we use the containers anyway and do some testing along the way then.

For now, I'd just leave this workaround in the recipe and we can remove it once the underlying bug is sorted out.